python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -148,3 +149,31 @@ def pot(self,
             model = Core().read_model(model_path)
             model.reshape(orig_shape)
         return model
+
+    def async_predict(self, inputs, jobs):
```

Comment:
any BKM for `job`? Can we set the job number according to OS environment(n_cores?) automatically? Otherwise, please give a default value to `jobs`.

Reply:
change `jobs` to `concurrency` or `parallelism?

Reply:
> change `jobs` to `concurrency` or `parallelism`?

The documentation of OpenVINO  doesn't mention that async infer requests in the `AsyncInferQueue` are performed in parallel, it just says `start_async` function will do async inference and return immediately without blocking until the inference finishes, so I think change to `concurrency` or `parallelism` may be a little inaccurate?
Since `jobs` means number of infer requests will be created in the `AsyncInferQueue`, how about change to something like `num_infer_requests` ?

Here are the releated docs:
https://docs.openvino.ai/latest/api/ie_python_api/_autosummary/openvino.runtime.AsyncInferQueue.html#openvino.runtime.AsyncInferQueue.start_async
https://docs.openvino.ai/latest/api/ie_python_api/_autosummary/openvino.runtime.InferRequest.html#openvino.runtime.InferRequest.start_async

Reply:
> any BKM for `job`? Can we set the job number according to OS environment(n_cores?) automatically? Otherwise, please give a default value to `jobs`.

OK. 
Since `jobs` defaults to 0 in definition of `AsyncInferQueue`, and if it's set to 0, the value will be automatically set by OpenVINO, so I think we can keep this default value.
https://docs.openvino.ai/latest/api/ie_python_api/_autosummary/openvino.runtime.AsyncInferQueue.html#openvino.runtime.AsyncInferQueue.__init__

Reply:
> `num_infer_requests`

just `num_requests`